packages/blockchain-wallet-v4-frontend/src/services/errors/NabuError/utils/isNabuErrorInNetworkResponse/isNabuErrorInNetworkResponse.ts

```diff
@@ -0,0 +1,18 @@
+import { NabuErrorInNetworkResponseUtility } from './isNabuErrorInNetworkResponse.types'
+
+const isNabuErrorInNetworkResponse: NabuErrorInNetworkResponseUtility = (payload) => {
+  const hasUxAttribute = Object.prototype.hasOwnProperty.call(payload, 'ux')
+
+  if (hasUxAttribute) {
```

Comment:
We can assume, if there is a ux attribute it will come with at least a title and message - this is the contract the nabu service is providing us

Reply:
@oliveratkinson-bc then it is safe to remove the `hasMessageInUx` and `hasTitleInUx` checks below.

will change this function to
```
const isNabuErrorInNetworkResponse: NabuErrorInNetworkResponseUtility =  (payload) => {
  return Object.prototype.hasOwnProperty.call(payload, 'ux');
}
```